app/src/main/java/com/duckduckgo/app/notification/model/EnableAppTpNotification.kt

```diff
@@ -0,0 +1,151 @@
+/*
+ * Copyright (c) 2023 DuckDuckGo
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.duckduckgo.app.notification.model
+
+import android.app.PendingIntent
+import android.content.Context
+import android.os.Bundle
+import com.duckduckgo.app.browser.R
+import com.duckduckgo.app.global.DispatcherProvider
+import com.duckduckgo.app.notification.NotificationRegistrar
+import com.duckduckgo.app.notification.TaskStackBuilderFactory
+import com.duckduckgo.app.notification.db.NotificationDao
+import com.duckduckgo.app.pixels.AppPixelName
+import com.duckduckgo.app.statistics.VariantManager
+import com.duckduckgo.app.statistics.isNextLevelPrivacyNotificationEnabled
+import com.duckduckgo.app.statistics.isOneEasyStepForPrivacyNotificationEnabled
+import com.duckduckgo.app.statistics.pixels.Pixel
+import com.duckduckgo.di.scopes.AppScope
+import com.duckduckgo.mobile.android.vpn.AppTpVpnFeature
+import com.duckduckgo.mobile.android.vpn.VpnFeaturesRegistry
+import com.duckduckgo.mobile.android.vpn.ui.onboarding.VpnOnboardingActivity
+import com.squareup.anvil.annotations.ContributesMultibinding
+import javax.inject.Inject
+import kotlinx.coroutines.CoroutineScope
+import kotlinx.coroutines.async
+import kotlinx.coroutines.runBlocking
+import timber.log.Timber
+
+class EnableAppTpNotification(
+    private val context: Context,
+    private val notificationDao: NotificationDao,
+    private val variantManager: VariantManager,
+    private val vpnFeaturesRegistry: VpnFeaturesRegistry,
+) : SchedulableNotification {
+
+    override val id = "com.duckduckgo.protection.enableapptp"
+
+    override suspend fun canShow(): Boolean {
+        if (notificationDao.exists(id)) {
+            Timber.v("Notification already seen")
+            return false
+        }
+
+        if (vpnFeaturesRegistry.isFeatureRegistered(AppTpVpnFeature.APPTP_VPN)) {
+            return false
+        }
+
+        if (!variantManager.isOneEasyStepForPrivacyNotificationEnabled() && !variantManager.isNextLevelPrivacyNotificationEnabled()) {
```

Comment:
There are 2 versions of the notification. We show the notification only if the app is in the expected variant.